test/scripts/run_e2e_tests.sh

```
@@ -4,7 +4,7 @@ SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
 source "${SCRIPT_DIR}"/functions
 
 REPORTS=${1}
-GO_E2E_DIRS=${@:2}
+GO_E2E_DIRS=(${@:2})
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)